### ISSUE TITLE
[FEATURE] Trace Question Answering models to TorchScript and Onnx format 

### ISSUE THREAD

**Issue by dhrubo-os** [opensearch-project/opensearch-py-ml#304]
We are planning to add more model supports in ML-Commons. https://github.com/opensearch-project/ml-commons/issues/1164

The target of this issue is to trace 3 popular pre-trained Question answering models to TorchScript and Onnx format. In this repo we traced pre-trained sentence embedding models into [torchscript](https://github.com/opensearch-project/opensearch-py-ml/blob/main/opensearch_py_ml/ml_models/sentencetransformermodel.py#L765) and [onnx](https://github.com/opensearch-project/opensearch-py-ml/blob/main/opensearch_py_ml/ml_models/sentencetransformermodel.py#L878)

We need to build the similar method to trace summarization models. Primarily we can target these models:

1. [distilbert-base-cased-distilled-squad](https://huggingface.co/distilbert-base-cased-distilled-squad)
2. [distilbert-base-uncased-distilled-squad](https://huggingface.co/distilbert-base-uncased-distilled-squad)
3. [bert-large-uncased-whole-word-masking-finetuned-squad](https://huggingface.co/distilbert-base-uncased-distilled-squad)

I created a feature branch : feature/summarization_model_conversation/. All the development of this issue should be done in that branch.

**Comment by faradawn** [comment]
Hi, I would like to tackle this issue.

I plan to create a similiar script called distilbert model besides Sentence Transformer:
```
opensearch-py-ml/opensearch_py_ml/ml_models/sentencetransformermodel.py
opensearch-py-ml/opensearch_py_ml/ml_models/distilbertmodel.py
```
Specifically, I plan to

1. Download a Distlled Bert model. 
2. Create a save_to_pt and to_onnx function utilizing torch.jit.trace.
3. In the future, create a train function if needed.

If there is anything I can do, please let me know.

**Comment by dhrubo-os** [comment]
Sure assigning to you.

**Comment by dhrubo-os** [comment]
May be we can create a question answering model class and work in there? distilbertmodel is one type of model, so I don't think we should create separate classes for `distilbertmodel`

**Comment by faradawn** [comment]
Hi @dhrubo-os,

Got it -- make sense! Will 1) create a question_answering.py model, and 2) raise PR in that branch!

Thanks,
Faradawn
